src/demo-app/grid-list/grid-list-demo.scss

```diff
@@ -1,5 +1,4 @@
 .demo-grid-list {
-  width: 1100px;
```

Comment:
I think that this one still needs some kind of width so it doesn't become too wide. This could be changed to a `max-width` so it doesn't break down on smaller screens.